algorithms/ar-radsrt/cpp/radix_sort.cpp

```diff
@@ -0,0 +1,48 @@
+#include<iostream>
+#include<list>
+#include<cmath>
+using namespace std;
+void display(int *array, int size) {
+   for(int i = 0; i<size; i++)
+      cout << array[i] << " ";
+   cout << endl;
+}
+void radixSort(int *arr, int n, int max) {
+   int i, j, m, p = 1, index, temp, count = 0;
```

Comment:
Codacy found an issue: [Variable 'p' is assigned a value that is never used.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6346241)

Reply:
Yeah,the variable p didn't require any assignment to 1,so it has now been removed